src/gitlab.js

```diff
@@ -90,7 +100,7 @@ export default class VerdaccioGitLab implements IPluginAuth {
     const cachedUserGroups = this._getCachedUserGroups(user, password);
     if (cachedUserGroups) {
       this.logger.debug(`[gitlab] user: ${user} found in cache, authenticated with groups:`, cachedUserGroups);
-      return cb(null, cachedUserGroups.publish);
+      return cb(null, [user]);
```

Comment:
I only return the username here, as I will be reading from the cache directly in the `allow_publish` and `allow_access` functions below.